.travis.yml

```diff
@@ -19,15 +19,46 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DMD=1.081.0 DIST=xenial F=production AFTER_SCRIPT=1
+        - DMD=1.081.0 DIST=xenial F=production
         - DMD=1.081.0 DIST=xenial F=devel
         - DMD=2.070.2.s12 DIST=xenial F=production
         - DMD=2.070.2.s12 DIST=xenial F=devel
         - DMD=2.071.2.s1 DIST=xenial F=production
         - DMD=2.071.2.s1 DIST=xenial F=devel
 
-install: beaver dlang install
+install: |
+    zcat "build/last/tmp/docker.img.gz" |
+        /usr/bin/time -f 'Load took %es\n' docker load -q || true
```

Comment:
Travis UI shows stage timings, what is this for?

Reply:
Just copy&paste from other parts where there is not the only command. I don't think I will keep up after debugging the options.